### DISCUSSION TITLE
新版本的聚类的折叠/扩展交互案例无法正常运行

### DISCUSSION THREAD

**Discussion by zhudafu** [antvis/G6#5268] [General]
新版本的**聚类的折叠/扩展交互**案例无法正常运行,只有[这个链接](https://antv-2018.alipay.com/zh-cn/g6/3.x/demo/case/cluster-simplify.html)上的代码可以正常运行
在新版案例中不能正常运行
https://g6.antv.vision/examples/case/graphDemos/#simplifyCluster
将旧版代码中的以下部分
`<script src="https://gw.alipayobjects.com/os/antv/pkg/_antv.g6-3.1.1/build/g6.js"></script>`
替换为新版后代码不能正常运行：
`<script src="https://gw.alipayobjects.com/os/lib/antv/g6/4.3.11/dist/g6.min.js"></script>
`

此外
`
if (!ge.get('sourceNode').get('visible') && sourceModel.cluster !== model.cluster) {
...
} else if (ge.get('targetNode').get('visible') && targetModel.cluster !== model.cluster) {
  ...
}           `
处，`else if`后应该改成`(!ge.get('targetNode').get('visible') && targetModel.cluster !== model.cluster)`
否则会出现点击聚类节点时，普通节点的边会被漏掉无法显示

**Comment by Aarebecca** [comment]
转至Issue： https://github.com/antvis/G6/issues/5269
